doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -0,0 +1,1081 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "00193e4f",
+   "metadata": {},
+   "source": [
+    "# Fine-tuning a Torch object detection model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd6ac669",
+   "metadata": {},
+   "source": [
+    "This tutorial explains how to fine-tune an object detection model using Torch and the\n",
+    "[Ray AI Runtime](air) (AIR).\n",
```

Comment:
when i view this in the notebook rendered on GitHub, the link for `Ray AI Runtime` is broken -- is this just a problem with the PR's rendered view, or is the link broken?

Reply:
Looks like this is just an issue with the GitHub render. Built the documentation and the link works okay.